model/src/main/kotlin/config/HttpFileStorageConfiguration.kt

```diff
@@ -37,5 +48,5 @@ data class HttpFileStorageConfiguration(
 )
 
 class MaskStringConverter : StdConverter<String, String>() {
-    override fun convert(value: String) = "***"
+    override fun convert(value: String) = if (value.isNotEmpty()) "***" else ""
```

Comment:
BTW, is this change really required? If so, why?

Reply:
It was suggested by @mnonnenmacher that the field is only masked if not empty and is not printed else. Although he meant this only for the postfix I did not want to create two converters and applied this for the headers, too.

Reply:
I see, so this is more or less a trick to get an empty string here that you don't need to serialize. This should be explained in the commit message.